presto-cli/src/test/java/io/prestosql/cli/TestCsvPrinter.java

```diff
@@ -93,6 +95,40 @@ public void testCsvPrintingNoHeader()
         assertEquals(writer.getBuffer().toString(), expected);
     }
 
+    @Test
+    public void testCsvPrintingNoRowsWithoutQuotes()
+            throws Exception
+    {
+        StringWriter writer = new StringWriter();
+        List<String> fieldNames = ImmutableList.of("first", "last");
+        OutputPrinter printer = new CsvPrinter(fieldNames, writer, NO_QUOTES);
+
+        printer.finish();
+
+        assertEquals(writer.getBuffer().toString(), "first,last\n");
+    }
+
+    @Test
+    public void testCsvPrintingNoHeaderWithoutQuotes()
+            throws Exception
+    {
+        StringWriter writer = new StringWriter();
+        List<String> fieldNames = ImmutableList.of("first", "last", "quantity");
+        OutputPrinter printer = new CsvPrinter(fieldNames, writer, NO_HEADER_AND_QUOTES);
+
+        printer.printRows(rows(
+                row("hello", "world", 123),
+                row("a", null, 4.5)),
+                          true);
+        printer.finish();
+
+        String expected = "" +
+                          "hello,world,123\n" +
+                          "a,,4.5\n";
```

Comment:
is this automated indentation or have you formatted that by yourself? Please use automated indentation.